src/Baballonia/App.axaml.cs

```diff
@@ -144,6 +145,12 @@ public override void OnFrameworkInitializationCompleted()
                 var defaultSettings = Path.Combine(AppContext.BaseDirectory, LocalSettingsService.DefaultLocalSettingsFile);
                 Directory.CreateDirectory(Path.GetDirectoryName(settingsLocation)!);
                 File.Copy(defaultSettings, settingsLocation);
+                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
```

Comment:
Please use OperatingSystem.IsLinux() here for consistency. (!Windows) will also effect calls on MacOS, we want this to be a Linux only operation.

Reply:
This problem could still happen on macOS

Reply:
Sure. Either way, if you could change the syntax, that would be great

Reply:
done